### ISSUE TITLE
Logs not printing to console on Flutter Web

### ISSUE THREAD

**Issue by jtromo** [DataDog/dd-sdk-flutter#485]
**Describe what happened**

Logging for Flutter Web is not properly printing to console. The log is sent correctly to DataDog and everything works in Android/iOS.

Imports are added to index.html
```
<script type="text/javascript" src="https://www.datadoghq-browser-agent.com/datadog-logs-v4.js"></script>
<script type="text/javascript" src="https://www.datadoghq-browser-agent.com/datadog-rum-slim-v4.js"></script>
```

Logger setup:
```
final logger = DatadogSdk.instance.createLogger(LoggingConfiguration(
    loggerName: "Splash",
    sendNetworkInfo: true,
    printLogsToConsole: true,
    sendLogsToDatadog: true,
    datadogReportingThreshold: Verbosity.verbose,
  ));
```
Button action: (verified is called via breakpoint/print)
```
logger.info("Test Flutter Web Info Log: Button Press");
print("Normal print: Button Press");
```

Console
```
Launching lib/main.dart on Chrome in debug mode...
main.dart:1
This app is linked to the debug service: ws://127.0.0.1:62197/Fa2tpsIvnYc=/ws
Debug service listening on ws://127.0.0.1:62197/Fa2tpsIvnYc=/ws
Connecting to VM Service at ws://127.0.0.1:62197/Fa2tpsIvnYc=/ws
The webOnlyWarmupEngine API is deprecated and will be removed in a future release. Please use `bootstrapEngine` from `dart:ui_web` instead.
The platformViewRegistry getter is deprecated and will be removed in a future release. Please import it from `dart:ui_web` instead.
The webOnlySetPluginHandler API is deprecated and will be removed in a future release. Please use `setPluginHandler` from `dart:ui_web` instead.
WARNING: found an existing <meta name="viewport"> tag. Flutter Web uses its own viewport configuration for better compatibility with Flutter. This tag will be replaced.
The debugEmulateFlutterTesterEnvironment getter is deprecated and will be removed in a future release. Please use `debugEmulateFlutterTesterEnvironment` from `dart:ui_web` instead.
Datadog Browser SDK:
Normal print: Button Press
```

**Steps to reproduce the issue:**
* Create logger with verbose Verbosity and printLogsToConsole true
* Print info log 
* The following appears to be printed to console without the log text: "Datadog Browser SDK:"

**Describe what you expected:**
* Logger should print log text to console.

**Additional context**
 - Datadog SDK version:
```
datadog_flutter_plugin: ^1.6.0
datadog_tracking_http_client: ^1.4.0
```

Flutter Doctor
```
[✓] Flutter (Channel stable, 3.13.1, on macOS 13.4.1 22F82 darwin-arm64, locale en-US)
    • Flutter version 3.13.1 on channel stable at /Users/jromo/Development/libraries/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision e1e47221e8 (5 weeks ago), 2023-08-22 21:43:18 -0700
    • Engine revision b20183e040
    • Dart version 3.1.0
    • DevTools version 2.25.0

[✓] Android toolchain - develop for Android devices (Android SDK version 33.0.2)
    • Android SDK at /Users/jromo/Library/Android/sdk
    • Platform android-33, build-tools 33.0.2
    • ANDROID_HOME = /Users/jromo/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jbr/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.15+0-b2043.56-8887301)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 14.3.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Build 14E300c
    • CocoaPods version 1.12.0

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2022.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.15+0-b2043.56-8887301)

[✓] VS Code (version 1.81.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.72.0

[✓] Connected device (2 available)
    • macOS (desktop)              • macos                                • darwin-arm64   • macOS 13.4.1 22F82 darwin-arm64
    • Chrome (web)                 • chrome                               • web-javascript • Google Chrome 117.0.5938.92

[✓] Network resources
    • All expected network resources are available.

• No issues found!
```

**Comment by fuzzybinary** [comment]
Hi @jtromo,

Thanks for reporting and sorry for the delay. We are in the process of reworking some of the Flutter Web logic and I'll make sure this is addressed as part of that.

Thanks!

**Comment by fuzzybinary** [comment]
Hi @jtromo,

Our new system for outputting logs to console should now be live in 2.0.0. Can you give it a try and see if it fixes your issue?

**Comment by fuzzybinary** [comment]
Hi @jtromo 

I'm going to close this. Please reopen if you have any issues.

**Comment by Den-creator** [comment]
@fuzzybinary is any option to disable printing logs to flutter console on iOS/Android ?

**Comment by baha-selmi12** [comment]
Hello I'm having the same issue is there a solution ?

**Comment by fuzzybinary** [comment]
@baha-selmi12 It might be worth opening a new bug with more information, how you're configuring the SDK and logger, and steps to reproduce if possible. This issue should have been solved with the release of 2.0.  If you're having an issue, it's likely unrelated.
